pkg/watchdog/init.sh

```diff
@@ -64,6 +75,8 @@ if [ -c /dev/watchdog ]; then
 else
     log "Platform has no /dev/watchdog"
     USE_HW_WATCHDOG=0
+    # in case of no hardware watchdog we must survive OOM for watchdog`s parent process, i.e. containerd-shim
+    adjust_parent_oom_score
```

Comment:
Sure, but wouldn't we want to do the adjustment even if we have hardware watchdog? Makes for less different cases to think about if all of the processes for the watchdog container have the same score.

Reply:
Why not, will do